### DISCUSSION TITLE
[DEPLOYMENT] Security patches

### DISCUSSION THREAD

**Discussion by bdoublet91** [keycloak/keycloak#33230] [Q&A] [answered]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

dist/quarkus

### Describe the bug

Hi,

With the new CVE of keycloak (CVE-2024-8698), we tried to upgrade minor version 22.0.13 to apply the security patches. This  version was tag in github and not released. (no more release version of 22.0.X since 22.0.5 one year ago)
I tried to build the 22.0.13 from the dockerfile (need to edit the source archive) but it doesn't build the code source, I need to deploy a full setup (java + maven) to build kc binaries and after create the docker image. Am I right ?
So I dont understand why you provide security fix in tag version barely impossible to deploy ? maybe create release with gthub workflow would be better for the community to deploy it ?

Thanks

### Version

22.0.5

### Regression

- [ ] The issue is a regression

### Expected behavior

Easy deploy latest security patches 22.0.13

### Actual behavior

Barely impossible or loss a lot of time ...

### How to Reproduce?

Try to deploy 22.0.13 in docker

### Anything else?

thanks

**Comment by jonkoops** [comment, marked as answer]
See #33148. If you want upstream support you'll have to use the latest version of Keycloak. The 22.x series is no longer supported.
